src/main/java/frc/robot/Robot.java

```diff
@@ -2,24 +2,35 @@
 
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.subsystems.BitBucketsSubsystem;
 import frc.robot.subsystems.climber.ClimberSubsystem;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Robot extends TimedRobot {
 
+    public static final List<BitBucketsSubsystem> ROBOT_SUBSYSTEMS = new ArrayList<>();
```

Comment:
There isn't any reason to make this a static final. final is good, but it shouldn't be static. It should also follow standard instance var naming conventions, i.e. robotSubsystems.

Reply:
I'm not sure how else it could work without being `static`, because I wanted to avoid the extra lines of code in `Robot.java` that were like `subsystems.add(climber)`
Unless there's an general instance of the Robot class that's used somewhere

Reply:
If each subsystem returns a list of its talons, then the robot can be responsible for collecting a list of talons by iterating each subsystem and asking for them. With a static var, the subsystem type itself is responsible for knowledge about all motors. It's a bit strange. If anything, we should have some other class (like the CANChecker) responsible for collecting motors from active subsystems and doing what it needs.

Ideally, a subsystem is as self contained as possible and only cares about its own motors and how to manipulate them.